WordPressKit/WordPressKit/BlogServiceRemoteREST+SiteSettings.swift

```diff
@@ -0,0 +1,48 @@
+import Foundation
+
+extension BlogServiceRemoteREST {
+
+    /// Fetch all Timezones from the API, with special handling for 'Manual Offsets' and 'UTC' sections
+    ///
+    /// - Parameters:
+    ///     - success The block that will be executed on success.
+    ///     - failure The block that will be executed on failure.
+    public func fetchTimeZoneList(success: @escaping ([String: [String: String]]) -> Void, failure: @escaping (Error) -> Void) {
+        let endpoint = "timezones"
+        let path = self.path(forEndpoint: endpoint, withVersion: ._2_0)
+
+        wordPressComRestApi.GET(path!, parameters: nil, success: { (response, _) in
+            let dict = response as? [String: AnyObject]
```

Comment:
I'd extract a helper method in the remote here that can pull the data we want from a response dictionary. Check out some of the other remotes, which have methods such as `remoteThemeFromDictionary:`.